### PR TITLE
Lesson 1.7

### PR DESCRIPTION
**Pull request by secch97**
- Deleted custom hook from lesson 1.6.
- Simulated async API fetching with setTimeOut  and useEffect hook.
- Added a loading state.
- Created a conditional loading indicator.
- Styled the conditional loading indicator.
- Disabled input and button when loading status is true. Enabled them when loading status is false.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -45,10 +60,20 @@ const App = () => {
...
+          {
+            isLoading ? 
+            (
+              <div className='loading-screen-container'>
+                <FontAwesomeIcon icon={["fas", "spinner"]} size="5x" spin className='loading-screen'/>
+              </div>
+            ) 
+            : 
+            (<TodoList todoList={todoList} onRemoveTodo={handleRemoveTodo}/>)
+          }
```
Comment (viniciuspietscher) [inline src/App.jsx:75]: Very cool loading spinner! :100:

File: src/App.jsx [hunk 2]
```diff
@@ -8,24 +8,39 @@ import { scrollToTop } from './helpers/scrollToTop';
...
+            todoList: JSON.parse(localStorage.getItem("savedTodoList")) ?? []
+          }
+        })
+      }, 2000);
+    });
+
+    todoListFetch.then((result) => {
+      setTodoList(result.data.todoList);
+      setIsLoading(false);
+    });
```
Comment (viniciuspietscher) [inline src/App.jsx:35]: Great job! :1st_place_medal: 

Another way of writing this is just creating a new promise without saving it into a variable
image

**Review by viniciuspietscher — APPROVED** [review]
Great work Saul! You're doing awesome! :1st_place_medal:

File: src/App.jsx (hunk 1 above)
Comment (secch97) [reply src/App.jsx:75]: Thank you Vinicius!

File: src/App.jsx (hunk 2 above)
Comment (secch97) [reply src/App.jsx:35]: Thanks for the advice! It saves lines of code by doing this. I will update my code to your suggestion!

**Pull request merged**